### ISSUE TITLE
SSL Cert Update?

### ISSUE THREAD

**Issue by crawfoal** [sorentwo/braintree-elixir#84]
[From Braintree](https://developers.braintreepayments.com/reference/general/best-practices/ruby):

> We will be updating our root SSL provider for API traffic to align with security standards starting April 1, 2020 in sandbox and May 1, 2020 in production. To avoid interruption to your processing, please update your SDK version to the appropriate minimum versions specified on this page.

Does this mean [the CA cert file in this repo](https://github.com/sorentwo/braintree-elixir/blob/master/priv/certs/api_braintreegateway_com.ca.crt) needs to be updated?

**Comment by sorentwo** [comment]
Possibly. I'll check an official repo to compare certs. Thanks for the notice!

**Comment by sorentwo** [comment]
tl;dr No update is needed, you're safe.

The `minimum version` that they reference for Ruby is 2.57, which uses the same cert bundle that we are (that's where this one came from). There is a newer certificate bundle available from 2.81, but it isn't compatible without other changes.

**Comment by crawfoal** [comment]
👍 Thank you for taking a look!
